core/server/data/export/index.js

```diff
@@ -23,10 +23,11 @@ exportFileName = function exportFileName() {
     var datetime = (new Date()).toJSON().substring(0, 10),
         title = '';
 
-    return settings.read(_.extend({}, {key: 'title'}, modelOptions)).then(function (result) {
+    return models.Settings.findOne(_.merge({key: 'title'}, modelOptions)).then(function (result) {
         if (result) {
-            title = serverUtils.safeString(result.settings[0].value) + '.';
+            title = serverUtils.safeString(result.attributes.value) + '.';
```

Comment:
Should we be moving towards using `.get('value')`, e.g. the more standard public API for bookshelf here? I know there are a few places in Ghost where we use `.attributes.x` however most of this is older code from before I understood how it worked 😬

Not a big deal just thinking that we should probably prefer one or the other?